app.js

```diff
@@ -6,6 +6,7 @@ var logger = require("morgan");
 var indexRouter = require("./routes/index");
 var patientsRouter = require("./routes/patients");
 let tasksRouter = require("./routes/tasks");
+var doctorsRouter = require("./routes/doctors");
```

Comment:
If you can try not to use `var` and instead use `let` or `const`. I forget to follow this rule sometimes. `let` & `const` is a good standard to follow with JS.